uri/uri.cpp

```diff
@@ -97,4 +101,63 @@ Uri Uri::parse(std::string uristr, std::optional<std::reference_wrapper<Uri cons
     return uri;
 }
 
+// https://w3c.github.io/FileAPI/#unicodeBlobURL
```

Comment:
The URL spec is coupled to the File API. The spec says that a URL has an associated blob URL entry. I saw that, and implemented blob URL generation as a separate thing, because two the concepts are related, and because I figure generating URLs should be a feature of a URL library. It's not part of URL parsing. I can get rid of it if we don't want it.